### ISSUE TITLE
SimplePromptCallback::onAuthenticationError doesn't return errorCode

### ISSUE THREAD

**Issue by GPVSensing** [SelfLender/react-native-biometrics#124]
@Override
    public void onAuthenticationError(int errorCode, @NonNull CharSequence errString) {
        super.onAuthenticationError(errorCode, errString);
        if (errorCode == BiometricPrompt.ERROR_NEGATIVE_BUTTON || errorCode == BiometricPrompt.ERROR_USER_CANCELED) {
            WritableMap resultMap = new WritableNativeMap();
            resultMap.putBoolean("success", false);
            resultMap.putString("error", "User cancellation");
            this.promise.resolve(resultMap);
        } else {
            **this.promise.reject(errString.toString(), errString.toString());**
            **Can we return errorCode also ??**
        }
    }

**Comment by blueish9** [comment]
yes, I've created a [PR](https://github.com/SelfLender/react-native-biometrics/pull/137) for that
